runtime/vm/resolvesupport.cpp

```diff
@@ -1979,6 +2070,55 @@ resolveMethodHandleRef(J9VMThread *vmThread, J9ConstantPool *ramCP, UDATA cpInde
 
 }
 
+j9object_t
+resolveOpenJDKInvokeHandle(J9VMThread *vmThread, J9ConstantPool *ramCP, UDATA cpIndex, UDATA resolveFlags)
+{
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+	j9object_t result = NULL;
+	bool canRunJavaCode = J9_ARE_NO_BITS_SET(resolveFlags, J9_RESOLVE_FLAG_JIT_COMPILE_TIME | J9_RESOLVE_FLAG_REDEFINE_CLASS);
+	J9RAMMethodRef *ramCPEntry = (J9RAMMethodRef*)ramCP + cpIndex;
+	UDATA invokeCacheIndex = ramCPEntry->methodIndexAndArgCount >> 8;
+	J9Class *ramClass = J9_CLASS_FROM_CP(ramCP);
+	j9object_t *invokeCache = ramClass->invokeCache + invokeCacheIndex;
+	result = *invokeCache;
```

Comment:
```suggestion
	j9object_t result = *invokeCache;
```

Reply:
Declare the local here and it only has 1 value - no need to reason about where the original NULL leaks to